src/index.ts

```diff
@@ -65,6 +65,12 @@ export class UltravoxTranscriptsChangedEvent extends UltravoxSessionStateChangeE
   }
 }
 
+export class UltravoxExperimentalMessageEvent extends Event {
```

Comment:
what is the thinking with calling this "Experimental"? does it have an expiration date and become something else? do we use "experimental" to denote things that may be deleted?